docker/Dockerfile

```diff
@@ -312,6 +310,9 @@ USER nautobot
 WORKDIR /opt/nautobot
 
 COPY --from=final-dev --chown=nautobot /opt/nautobot/nautobot_config.py /opt/nautobot/nautobot_config.py
+RUN mkdocs build --strict=False -q
```

Comment:
Why `--strict=False`?

Reply:
Need to fix, I had other issues and was trying to get around it.